build.gradle

```diff
@@ -76,8 +76,8 @@ allprojects {
 
     configurations.all {
         resolutionStrategy.eachDependency { details ->
-            if (details.requested.group == 'io.dropwizard.metrics' && details.requested.name != 'metrics-jmx') {
-                details.useVersion '3.2.6'
+            if (details.requested.group == 'io.dropwizard.metrics') {
+                details.useVersion '4.1.1'
```

Comment:
Is this entire resolution strategy block necessary anymore?

Reply:
Not entirely, I removed it from tritium, but figured this project is more likely to push the version up unexpectedly. That said, it's probably fine if that happens, and we can add constraints if we encounter failing cases.